### PR TITLE
LRDOCS-3522

### PR DESCRIPTION
**Pull request by justinchoi001**

### CODE REVIEW

File: discover/deployment/articles-dxp/01-installing-liferay/07-installing-liferay-on-tcserver.markdown [hunk 1]
```diff
@@ -1,9 +1,6 @@
 # Installing @product@ on tc Server [](id=installing-liferay-on-tc-server)
 
-The easiest way to get @product@ running on tc Server is to [download a bundle](https://web.liferay.com/group/customer/downloads/portal/product). If
-that's not an option, you can install @product@ onto tc Server. In addition to a
-supported version of [tc Server](https://network.pivotal.io/products/pivotal-tcserver) (version 3 or
-higher), make sure you have acquired the following items (from [Liferay's customer portal](https://web.liferay.com/group/customer/dxp/downloads/digital-enterprise))
+@product@ can be installed onto tc Server. In addition to a supported version of [tc Server](https://network.pivotal.io/products/pivotal-tcserver) (version 3.1.x), make sure you have acquired the following items (from [Liferay's customer portal](https://web.liferay.com/group/customer/dxp/downloads/digital-enterprise))
```
Comment (sez11a) [inline discover/deployment/articles-dxp/01-installing-liferay/07-installing-liferay-on-tcserver.markdown]: We try never to put version numbers in the docs (docs become hard to maintain as version numbers change). Instead, provide a link to the support matrix.

File: discover/deployment/articles-dxp/01-installing-liferay/07-installing-liferay-on-tcserver.markdown [hunk 2]
```diff
@@ -31,35 +26,37 @@ $$$
...
 
-@product@ depends on some additional JARs that aren't included with tc Server by
-default. There are even more JARs that you'd find in a @product@ bundle
-that are not required but can be useful. If you don't have a @product@
-bundle, you can download the required JARs from third parties, as
-described below.
+@product depends on some additional JARs that are not included with tc Server by
+@default. There are even more JARs found in a product bundle that are not
+@required but can be useful. If you don't have a product bundle, you can
+@download the required JARs from third parties, as described below.
```
Comment (sez11a) [inline discover/deployment/articles-dxp/01-installing-liferay/07-installing-liferay-on-tcserver.markdown]: What happened here with the @ symbols? Editor malfunction? :-) This whole change should be reverted.

File: discover/deployment/articles-dxp/01-installing-liferay/07-installing-liferay-on-tcserver.markdown [hunk 3]
```diff
@@ -31,35 +26,37 @@ $$$
...
 +$$$
 
-**Note:** Many required and useful JARs are pre-installed when you build @product@ from the source code or [download a @product@ bundle](https://www.liferay.com/downloads/). If you want to acquire all of the
-JARs that ship with a @product@ bundle quickly, using one of these sources might
-save you time.
+**Note: Many required and useful JARs are pre-installed when you build @product@
+**from the source code or [download a @product@
+**bundle](https://web.liferay.com/group/customer/dxp/downloads/digital-
+**enterprise). If you want to acquire all of the JARs that ship with a @product@
+**bundle quickly, using one of these sources might save you time.
```
Comment (sez11a) [inline discover/deployment/articles-dxp/01-installing-liferay/07-installing-liferay-on-tcserver.markdown]: Same thing here. Two stars (\*\*) in Markdown make something bold. Something weird happen with your editor when you reformatted the paragraph, I think.

File: discover/deployment/articles-dxp/01-installing-liferay/07-installing-liferay-on-tcserver.markdown [hunk 4]
```diff
@@ -31,35 +26,37 @@ $$$
...
+One JAR definitely needed but not included in the dependencies zip is
+the database driver. Database drivers for *MySQL* and *PostgreSQL* can be found
 in a @product@ bundle or in the @product@ source code.
 
-There are several other dependency JARs that aren't included in the dependencies
-zip. If you don't have them already on hand or have access to a @product@ bundle,
-you'll have to download them yourself.
+There are several other dependency JARs that are not included in the dependencies
+zip. If you do not have them already on hand or have access to a @product@ bundle,
+they are available for download:
```
Comment (sez11a) [inline discover/deployment/articles-dxp/01-installing-liferay/07-installing-liferay-on-tcserver.markdown]: Please revert this change. We use contractions on purpose as a style.

File: discover/deployment/articles-dxp/01-installing-liferay/07-installing-liferay-on-tcserver.markdown (hunk 2 above)
Comment (justinchoi001) [inline discover/deployment/articles-dxp/01-installing-liferay/07-installing-liferay-on-tcserver.markdown]: I am not sure what happened. Reverting the changes.

**Comment by sez11a** [comment]
View just my changes: https://github.com/sez11a/liferay-docs/compare/justinchoi001:0f883b5d2d...4d58cbbf38
View total diff: https://github.com/sez11a/liferay-docs/compare/53a671fd45...4d58cbbf38
